core-web/libs/dotcms-scss/angular/dotcms-theme/components/_table.scss

```diff
@@ -8,25 +13,278 @@ p-table {
+            background-color: $color-palette-gray-200;
+            border: 1px solid $color-palette-gray-300;
+            border-width: 0 0 1px 0;
+            color: $black;
+            font-size: $font-size-xsm;
+            font-weight: $font-weight-semi-bold;
+            padding: $spacing-1;
+            text-align: left;
+            transition: none;
+            text-wrap: nowrap;
+
+            &:first-child {
+                padding-left: $spacing-3;
+            }
+
+            &:last-child {
+                padding-right: $spacing-3;
             }
         }
 
-        .p-datatable-tbody {
-            > tr:hover {
+        & .p-datatable-tfoot > tr > td {
+            text-align: left;
+            padding: $spacing-2 $spacing-2;
+            border: 1px solid $table-border-color;
+            border-width: 0 0 1px 0;
+            font-weight: $font-weight-semi-bold;
+            color: $black;
+            background: $white;
+        }
+
+        & .p-sortable-column {
+            outline: 0 none;
+
+            & .p-sortable-column-icon {
+                display: inline-block;
+                color: $color-palette-gray-700;
+                margin-left: $spacing-1;
+            }
+
+            & .p-sortable-column-badge {
+                border-radius: 50%;
+                height: 1.143rem;
+                min-width: 1.143rem;
+                line-height: 1.143rem;
+                color: $color-palette-primary;
+                background: $bg-highlight;
+                margin-left: $spacing-1;
+            }
+
+            &:not(.p-highlight):hover {
+                color: $text-color-hover;
+
+                & .p-sortable-column-icon {
+                    color: $black;
+                }
+            }
+
+            &.p-highlight {
+                & .p-sortable-column-icon {
+                    color: $color-palette-primary-500;
+                }
+            }
+        }
+
+        & .p-datatable-tbody > tr {
+            background: $white;
+            color: $black;
+            height: auto;
+            transition: none;
+            outline: 0 none;
+            cursor: pointer;
+            overflow-wrap: anywhere;
+
+            &:hover {
                 background-color: $color-palette-primary-200;
             }
+
+            & > td {
+                text-align: left;
+                border: 1px solid $color-palette-gray-300;
+                border-width: 0 0 1px 0;
+                padding: 0.625rem $spacing-1;
+                font-size: $font-size-smd;
+
+                &:first-child {
+                    padding-left: $spacing-3;
+                }
+
+                &:last-child {
+                    padding-right: $spacing-3;
+                }
+
+                & .p-row-toggler,
+                & .p-row-editor-init,
+                & .p-row-editor-save,
+                & .p-row-editor-cancel {
+                    width: $spacing-5;
+                    height: $spacing-5;
```

Comment:
spacing vars are for spacings not sizing.

Reply:
changed, however could not find reference of these classes in primeng code, leaving the rules to match primeng template, for now.